src/core/markdown.js

```diff
@@ -88,11 +88,15 @@ const inlineElems = new Set([
 
 class Renderer extends marked.Renderer {
   code(code, language, isEscaped) {
-    if (language === "webidl") {
+    if (this.isWebidlLanguageTag(language)) {
       return `<pre class="idl">${code}</pre>`;
     }
     return super.code(code, language, isEscaped);
   }
+
+  isWebidlLanguageTag(tag) {
+    return /(^webidl$|^idl$)/i.test(tag);
```

Comment:
The `idl` is common, so `web` can be treated as optional
```suggestion
    return /(^(web)?idl$)/i.test(tag);
```